src/components/VenueList/VenueCard/VenuePriceOverlay.jsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { PriceDisplay } from './PriceDisplay';
+import { LocationDisplay } from './LocationDisplay';
+
+const StyledBox = styled(Box)(({ theme }) => ({
+  position: 'absolute',
+  bottom: 0,
+  left: 0,
+  padding: theme.spacing(1),
+  backgroundColor: 'rgba(0,0,0,0.7)',
```

Comment:
this color is used in multiple places. Please create a variable in theme and use it